.github/workflows/build.yml

```diff
@@ -104,4 +94,4 @@ jobs:
         run: esy install --verbose
 
       - name: Build odoc
-        run: esy odoc --version
+        run: esy x dune exec src/odoc/bin/main.exe -- --version
```

Comment:
This doesn't seem right. I managed to make it work with this: https://github.com/Julow/odoc/commit/cd8fb9b28fe1e5fe7febb22266731c7b05507e74 ([esy build result](https://github.com/Julow/odoc/runs/1911160080))

Reply:
That looks much better - thanks!